community/kernel/src/main/java/org/neo4j/kernel/impl/transaction/command/Command.java

```diff
@@ -313,30 +247,45 @@ public long getRelId()
 
     public static abstract class TokenCommand<RECORD extends TokenRecord> extends Command
```

Comment:
Yes it is needed. `TokenRecord` is an `AbstractRecord` whilst the, e.g., `NodeRecord` is an `Abstract64bitsRecord`